plugins/ocm/src/components/ClusterStatusPage/ClusterStatusPage.tsx

```diff
@@ -173,10 +172,22 @@ const CatalogClusters = () => {
             title: 'Name',
             field: 'name',
             highlight: true,
+            customSort: (a, b) => {
+              return a.name.props.children.localeCompare(
+                b.name.props.children,
+                'en',
+              );
+            },
```

Comment:
Hi @Zaperex can you move the columns to a separate file and also create a type for data that each table row represents so that you dont have to type-cast params manually in the `customSort` function ? Something like:
```
         export const columns: TableColumn<RowType>[] = [{
            title: 'Name',
            field: 'name',
            highlight: true,
            customSort: (a: RowType, b: RowType) => {
              return a.name.props.children.localeCompare(
                b.name.props.children,
                'en',
              );
            },
          {
            title: 'Status',
            field: 'status',
            customSort: (a: RowType, b: RowType) => {
              const availabilityA = a.status.props.status.available;
              const availabilityB = b.status.props.status.available;
              if (availabilityA === availabilityB) return 0;
              return availabilityA ? -1 : 1;
            },
          },
           ...
         ]
```

Reply:
Just noting it down here that each row is actually the following type. I'll see if I can get the correct types.
```tsx
{
    name: React.JSX.Element;
    status: React.JSX.Element;
    infrastructure: string;
    version: React.JSX.Element;
    nodes: React.JSX.Element;
}
```